src/Cache/Engine/FileEngine.php

```diff
@@ -319,25 +319,31 @@ protected function _clearDirectory(string $path): void
      *
      * @param string $key The key to decrement
      * @param int $offset The number to offset
-     * @return void
+     * @return false
      * @throws \LogicException
      */
-    public function decrement(string $key, int $offset = 1): void
+    public function decrement(string $key, int $offset = 1)
     {
         throw new LogicException('Files cannot be atomically decremented.');
+
+        // phpcs:ignore
+        return false;
```

Comment:
Why is this necessary? Seems like a phpstan issue.

Reply:
Docblock of CacheEngineInterface::decrement() says it should return a boolean or int, so phpstan is unhappy if there's no retuy value.

But yes logically return value should not be necessary due to the exception. I can revert and ignore the error instead.

Reply:
We should Report this as phpstan bug. It clearly is.

Reply:
Turned out the only problem was the `void` return type hint.